src/coreComponents/constitutive/fluid/SingleFluidBase.hpp

```diff
@@ -178,15 +181,16 @@ class SingleFluidBase : public ConstitutiveBase
   arrayView2d< real64 const > dDensity_dPressure() const { return m_dDensity_dPressure; }
 
   arrayView2d< real64 const > initialDensity() const { return m_initialDensity; }
+  arrayView2d< real64 const > density_n() const { return m_density_n; }
 
   arrayView2d< real64 > viscosity() { return m_viscosity; }
   arrayView2d< real64 const > viscosity() const { return m_viscosity; }
 
   arrayView2d< real64 > dViscosity_dPressure() { return m_dViscosity_dPressure; }
   arrayView2d< real64 const > dViscosity_dPressure() const { return m_dViscosity_dPressure; }
```

Comment:
Do we still need `non const` accessors?  I think these properties are now always updated inside the constitutive models. Aren't they? We could try to do this in a separate PR though.

Reply:
Yes you are right they are useless, I removed them the non `const` accessors